backend/app/tests/seed_data/seed_data.py

```
@@ -0,0 +1,459 @@
+        project = session.exec(
+            select(Project).where(
+                Project.name == document_data.project_name,
+                Project.organization_id == organization.id,
+            )
+        ).first()
+        if not project:
+            raise ValueError(
+                f"Project '{document_data.project_name}' not found in organization '{organization.name}'"
+            )
+
+        users = session.exec(
```

Ruff (0.14.8)

Local variable `users` is assigned to but never used

Remove assignment to unused variable `users`

(F841)